plugins/nf-google/src/main/nextflow/cloud/google/batch/GoogleBatchTaskHandler.groovy

```diff
@@ -139,8 +140,21 @@ class GoogleBatchTaskHandler extends TaskHandler {
             .addAllCommands( ['/bin/bash','-o','pipefail','-c', cmd.toString()] )
             .addAllVolumes( launcher.getContainerMounts() )
 
+        if( task.config.getAccelerator() )
+            container
+                .addVolumes('/var/lib/nvidia/lib64:/usr/local/nvidia/lib64')
+                .addVolumes('/var/lib/nvidia/bin:/usr/local/nvidia/bin')
```

Comment:
Even if nvidia is the only one supported, it may be better to add those (and the privilege flag) only if the accelerator type is actually nvidia. What's the expected `type` pattern?

Reply:
All Nvidia accelerator type start with NVIDIA_
so we can add task.config.getAccelerator().getType().toLowerCase().startswith('nvidia_') to the if statement

Reply:
```suggestion
        if( task.config.getAccelerator() && task.config.getAccelerator().getType()?.toLowerCase().startsWith('nvidia-') )
            container
                .addVolumes('/var/lib/nvidia/lib64:/usr/local/nvidia/lib64')
                .addVolumes('/var/lib/nvidia/bin:/usr/local/nvidia/bin')
```